React/Views/RCTViewKeyboardEvent.m

```diff
@@ -28,48 +28,48 @@ + (instancetype)keyDownEventWithReactTag:(NSNumber *)reactTag
                                                   viewTag:reactTag
                                                      body:@{ @"capsLockKey" : @(capsLockKey),
                                                                 @"shiftKey" : @(shiftKey),
-                                                              @"controlKey" : @(controlKey),
-                                                               @"optionKey" : @(optionKey),
-                                                              @"commandKey" : @(commandKey),
+                                                                 @"ctrlKey" : @(controlKey),
+                                                                  @"altKey" : @(optionKey),
+                                                                 @"metaKey" : @(commandKey),
                                                            @"numericPadKey" : @(numericPadKey),
                                                                  @"helpKey" : @(helpKey),
                                                              @"functionKey" : @(functionKey),
-                                                            @"leftArrowKey" : @(leftArrowKey),
-                                                           @"rightArrowKey" : @(rightArrowKey),
-                                                              @"upArrowKey" : @(upArrowKey),
-                                                            @"downArrowKey" : @(downArrowKey),
+                                                               @"ArrowLeft" : @(leftArrowKey),
+                                                              @"ArrowRight" : @(rightArrowKey),
+                                                                 @"ArrowUp" : @(upArrowKey),
+                                                               @"ArrowDown" : @(downArrowKey),
```

Comment:
These probably shouldn't be here anymore as the `key` property now correctly represents them.